components/access-management/kfam/bindings_test.go

```diff
@@ -0,0 +1,72 @@
+package kfam
+
+import (
+	"testing"
+
+	rbacv1 "k8s.io/api/rbac/v1"
+)
+
+// Building Binding Object from k8s.io/api/rbac/v1
+func getBindingObject(binding string) *Binding {
+
+	return &Binding{
+		User: &rbacv1.Subject{
+			Kind: "user",
```

Comment:
nit: Can you use the `rbacv1.UserKind` constant instead?
It's a best practice to use the constants provided by the package, as any changes to the constant will be immediately reflected in our code.

Reply:
Changed